src/meltano/cli/add.py

```diff
@@ -47,11 +54,6 @@
     is_flag=True,
     help="Add a custom plugin. The command will prompt you for the package's base plugin description metadata.",
 )
-@click.option(
-    "--include-related",
-    is_flag=True,
-    help="Also add transform plugins related to the identified discoverable extractor.",
-)
```

Comment:
TODO: We may want to call out in migration docs somewhere that the `--include-related` CLI arg is removed from `meltano add`.